website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -784,9 +792,9 @@ The refined ORCA inventory catalog build script is available for download <a hre
 ### Addressing open questions
 
 - What is the proper date to use when receiving a date range from Cumulus for a reconciliation report?
-  - At this time, we are not going to include granule acquisition date since we cannot get it natively from cumulus but might add it in the future.
+  - After discussion with the Cumulus team, we have decided to use `cumulus_createdAt` timestamp.
 - Should granule acquisition date be a part of the metadata?
-  -  At this time, we are not going to include granule acquisition date since we cannot get it natively from Cumulus but might add it in the future.
+  -  After discussion with the Cumulus team, we have decided to use `cumulus_createdAt` timestamp.
```

Comment:
I would revert to the original comment since that is still valid. Acquisition date may be used for comparisons in the future but per the original comment it cannot be used.